src/js/themes/base.js

```diff
@@ -1049,6 +1049,13 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         },
         // any drop props
       },
+      group: {
+        container: {
+          pad: {
```

Comment:
Can we put the `border : { side: 'top' }` here?

But, I can see where this doesn't work so well as we don't want that on the first group. I'm wondering if we should structure this more as `group.separator: { color, size }` ?

Reply:
Agree, I was wondering the same and like that idea. Also, I've seen other grouped menus implement the separator as its own `div` all together as opposed to using border on one side of the box. I went the top border approach to try to avoid adding another div to the DOM -- are you good with that?

Reply:
Done.